src/Raven.Studio/typescript/viewmodels/resources/resources.ts

```diff
@@ -348,6 +371,20 @@ class resources extends viewModelBase {
         app.showBootstrapDialog(createDbView);
     }
 
+    activateResource(rsInfo: resourceInfo) {
+        let resource = this.resourcesManager.resources().filter(x => x.qualifiedName === rsInfo.qualifiedName)[0];
+        if (!resource)
+            return;
+
+        activeResourceTracker.default.resource(resource);
+        resource.activate();
```

Comment:
this call should trigger EVENT.Resource.Activate and update activeResourceTracker.default.resource(resource);, so please remove previous line.